src/coreclr/tools/Common/Compiler/DependencyAnalysis/CorInfoHelpers.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Internal.JitInterface;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    public class CorInfoHelpers
+    {
+        public static ReadyToRunHelperId getRunHelperIdFromHelperFunc(CorInfoHelpFunc helper)
```

Comment:
Naming nit: our conventions use PascalCase and the name seems a bit more general than it should be, since it only deals with static bases. My suggestion would be something like `GetReadyToRunHelperFromStaticBaseHelper`.

(The exceptions to the PascalCase rule is due to the fact that the parts of crossgen2/ilc that deal with the JIT-EE interface boilerplate are auto generated)